var/spack/repos/builtin/packages/pumi/package.py

```diff
@@ -27,7 +27,7 @@ class Pumi(CMakePackage):
     # to the added instability. The spack version string is 'develop' since
     # it compares greater than a numbered version (e.g., 2.1.0). The spack
     # version string 'master' compares less than a numbered version.
-    version('develop', branch='master')
+    version('develop', submodules=True, branch='master')
```

Comment:
```suggestion
    version('master', submodules=True, branch='master')
```

Reply:
Is the following statement no longer true?

```
    # The spack version string is 'develop' since
    # it compares greater than a numbered version (e.g., 2.1.0). The spack
    # version string 'master' compares less than a numbered version.
```

https://github.com/SCOREC/spack/blob/00845e1fc2c4f20d87b37d2992cd4fa6f62b958f/var/spack/repos/builtin/packages/pumi/package.py#L27-L29

Reply:
Yep, Spack now treats master and develop as both being greater than any numerical release. See #1983.

Reply:
Thank you.